plugins/module_utils/network/vyos/facts/ospfv2/ospfv2.py

```diff
@@ -159,6 +162,29 @@ def parse_leaf_list(self, conf, attrib):
                 lst.sort()
         return lst
 
+    def parse_passive(self, conf, attrib):
+        """
+        This function forms the regex to fetch the listed attributes
+        from the configuration data
+        :param conf: configuration data
+        :param attrib: attribute name
+        :return: generated rule list configuration
+        """
+        lst = []
+        items = []
+        if LooseVersion(get_os_version(self._module)) >= LooseVersion("1.4"):
+            if attrib == "passive-interface-exclude":
+                items = findall(r"^interface" + " (?:'*)(\\S+)(?:'*) passive disable$", conf, M)
+            else:
+                items = findall(r"^interface" + " (?:'*)(\\S+)(?:'*) passive$", conf, M)
+        # else:
```

Comment:
Please remove this comment, since this isn't functionally an 'else'. I'm assuming this is a left-over from a test?